sdk/servicebus/azure-servicebus/azure/servicebus/_common/auto_lock_renewer.py

```diff
@@ -163,6 +218,18 @@ def _auto_lock_renew(
                         # Renewable is a message
                         receiver.renew_message_lock(renewable)  # type: ignore
                 time.sleep(self._sleep_time)
+            if self._renewable(renewable):
```

Comment:
is there a reason there are 2 `if self._renewable(renewable):` lines? (other at 195). can the below be moved into the block above?

Reply:
yea, I think you're right, it should be moved into the block above